client/angular/services/adminDashDataService.js

```diff
@@ -108,13 +108,13 @@
         }
 
         function getSpeakers() {
-            return $http.get("/api/speakers").then(function (result) {
+            return $http.get("/admin/speakers").then(function (result) {
```

Comment:
I would keep the get on /api in case we want the client to process tweets according to the list of speakers.